src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTests.cs

```diff
@@ -9798,7 +9798,9 @@ .maxstack  4
 ");
         }
 
-        [Fact, WorkItem(538246, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/538246"), WorkItem(543655, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/543655")]
+        [ConditionalFact(typeof(WindowsDesktopOnly), Reason = "https://github.com/dotnet/roslyn/issues/33564")] // PROTOTYPE(DefaultInterfaceImplementation): https://github.com/dotnet/roslyn/issues/33564
```

Comment:
As mentioned here: https://github.com/dotnet/roslyn/issues/33564#issuecomment-466126580, you should be able to make these all work on both full framework and .netcore by using `G15` as the format specifier (which is what `ToString()` used to default to internally).

Reply:
I prefer to not spend time on this now since I cannot reliably verify the behavior locally, among other reasons.